src/hotspot/cpu/aarch64/globals_aarch64.hpp

```diff
@@ -93,6 +93,8 @@ define_pd_global(intx, InlineSmallCode,          1000);
           "Use SIMD instructions in generated array equals code")       \
   product(bool, UseSimpleArrayEquals, false,                            \
           "Use simpliest and shortest implementation for array equals") \
+  product(bool, UseSimpleStringEquals, true,                            \
```

Comment:
Do we really need a user-facing toggle, especially a product one? Under what situations do we expect the user to change this? It's useful for comparison but if the new implementation if demonstrably better then we should just delete the old one.

Reply:
True. But we've a way to go; taking this out should be the last setp.